uefi-raw/src/protocol/network/tcpv4.rs

```diff
@@ -0,0 +1,513 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+//! TCPv4 Protocol
+//!
+//! This module provides the TCPv4 Protocol interface definitions. The
+//! TCPv4 Protocol provides services to send and receive data streams
+//! over IPv4 networks.
+//!
+//! The protocol is defined in the [UEFI Specification, Section 28.1](https://uefi.org/specs/UEFI/2.11/28_Network_Protocols_TCP_IP_and_Configuration.html#efi-tcpv4-protocol).
+
+use crate::protocol::network::snp::NetworkMode;
+use crate::{Boolean, Event, Guid, Handle, Ipv4Address, Status, guid, newtype_enum};
+use core::ffi::c_void;
+use core::fmt::{Debug, Formatter};
+
+#[derive(Debug)]
+#[repr(C)]
+pub struct Tcpv4Protocol {
```

Comment:
nit: match the spec name for this protocol

```suggestion
pub struct Tcp4Protocol {
```

Same for all the other types like `Tcpv4ConnectionState`; it should match the name in the spec except for converting to PascalCase.

Reply:
good catch